Services/Notifications/templates/default/notifications.js

```diff
@@ -1,17 +1,7 @@
 var OSDNotifier, OSDNotifications = settings => {
-    const createThrottle = timeout => {
-        let clear = () => {};
-        return callback => {
-            clear();
-            clear = clearTimeout.bind(window, setTimeout(callback, timeout));
-        };
-    };
+	let lastRequest = 0;
```

Comment:
Please move this line into the `poll`  function (After line 54) as it is no where else needed.

Please change this file to use either only spaces or only tabs. Currently they are both in use.